### ISSUE TITLE
Package as snap

### ISSUE THREAD

**Issue by kspearrin** [bitwarden/cli#6]
Distribute the CLI as a snap

**Comment by kspearrin** [comment]
@popey Do you have any experience with packaging Node.js apps as snaps? We'd like to distribute this CLI for Bitwarden on the snap store too but I am having a hard time understanding how this is suppose to work.

Following the docs here: https://docs.snapcraft.io/build-snaps/node

I have created the following `snapcraft.yaml`: https://github.com/bitwarden/cli/blob/master/stores/snap/snapcraft.yaml

Running `snapcraft` on this seems to just install [`@bitwarden/cli` globally from NPM](https://www.npmjs.com/package/@bitwarden/cli). That seems kind of odd. We want the snap package to be self contained. It shouldn't use the host machine's NPM at all. Should we just try to package [our Linux binary](https://github.com/bitwarden/cli/releases/tag/v1.0.0) instead?

**Comment by kspearrin** [comment]
@popey I gave up on the node version and just went for an easy binary dump instead. Now updated at https://github.com/bitwarden/cli/blob/master/stores/snap/snapcraft.yaml

Another question: we'd like for the CLI to be available with the `bw` command (how it is used everywhere else and in our docs). As it stands, snap users would have to use `bitwarden-cli.bw` instead. I noticed that it says we can request for the `bw` command on the snap forums. Is it likely that we will be granted this short command or should we just forget trying to allow users to use `bw` and keep `bitwarden-cli` (name of package) as the command instead?
